plugins/outputs/cloudwatch/cloudwatch_test.go

```diff
@@ -401,14 +400,14 @@ func TestWriteError(t *testing.T) {
 	}
 	cloudWatchOutput.Write(metrics)
 
-	var sum float64
+	// Sum time for all retries.
+	var sum int
 	for i := 0; i < defaultRetryCount; i++ {
-		sum = sum + math.Pow(2, float64(i))
+		sum += 1 << i
 	}
-	time.Sleep(time.Duration(backoffRetryBase*int64(sum)) * time.Millisecond)
-
+	time.Sleep(backoffRetryBase * time.Duration(sum))
 	assert.True(t, svc.AssertNumberOfCalls(t, "PutMetricData", 5))
-
+	cloudWatchOutput.Close()
```

Comment:
No real change in `TestWriteError()`.
The type of `backoffRetryBase` was changed elsewhere.
The extra call to `cloudWatchOutput.Close()` is just for cleanup.